--- HEAD:Registry/LMSOFTWAREKUKA ROBOTER GMBHKUKA BOFOCX Control'sFileHandler.amr
+++ backup:Registry/LMSOFTWAREKUKA ROBOTER GMBHKUKA BOFOCX Control'sFileHandler.amr
@@ -1,6 +1,6 @@
 [\LocalMachine\SOFTWARE\KUKA ROBOTER GMBH\KUKA BOF\OCX Control's\FileHandler]
 "SpecialDrives"=
-"StartUpPath"="KRC:\R1\Program\Styles\optiones\"
+"StartUpPath"="KRC:\R1\"
 
 
 [\LocalMachine\SOFTWARE\KUKA ROBOTER GMBH\KUKA BOF\OCX Control's\FileHandler\Display]
